mongoid.gemspec

```diff
@@ -29,10 +29,14 @@ Gem::Specification.new do |s|
     warn "[#{s.name}] Warning: No private key present, creating unsigned gem."
   end
 
-  s.required_ruby_version     = ">= 2.3"
+  s.required_ruby_version     = ">= 2.5"
```

Comment:
I think this needs to be reverted as well, since we are going to release this diff on all stable branches.